samples/nrf9160/nrf_cloud_mqtt_multi_service/README.rst

```diff
@@ -185,11 +194,54 @@ Configuration
 *************
 |config|
 
-Configuration options
-=====================
+Disabling key features
+======================
+
+The following key features of this sample may be independently disabled:
+
+* GNSS-based location tracking
+* Cellular-based location tracking
+* Temperature tracking
+* GNSS assistance (A-GPS)
+* Predictive GNSS assistance (P-GPS)
+
+To disable GNSS-based location tracking, set the ref:`CONFIG_LOCATION_TRACKING_GNSS <CONFIG_LOCATION_TRACKING_GNSS>` option to disabled.
+
+To disable cellular-based location tracking, set the ref:`CONFIG_LOCATION_TRACKING_CELLULAR <CONFIG_LOCATION_TRACKING_CELLULAR>` option to disabled.
+
+Disabling both options will completely disable location tracking.
+
+To disable temperature tracking, set the ref:`CONFIG_TEMP_TRACKING <CONFIG_TEMP_TRACKING>` option to disabled.
+
+To disable A-GPS, set the :kconfig:option:`CONFIG_NRF_CLOUD_AGPS` option to disabled.
+
+To disable P-GPS, set the :kconfig:option:`CONFIG_NRF_CLOUD_PGPS` option to disabled.
+
+
+note::
+  MQTT should only be used with applications that need to stay connected constantly or transfer data frequently.
+  While this sample does allow its core features to be slowed or completely disabled, in real-world applications, you should carefully consider your data throughput and whether MQTT is an appropriate solution.
+  If you find yourself wanting to disable or excessively slow any of these features for a real-world application, other solutions, such as the `nRF Cloud Rest API`_, may be more appropriate.
```

Comment:
This is good.  Small nit is that it should say `excessively slow all of these features` -- if any of them are still frequent, that's sufficient to justify MQTT.